rotkehlchen/chain/ethereum/oracles/uniswap.py

```diff
@@ -279,6 +283,7 @@ class UniswapV3Oracle(UniswapOracle):
     def __init__(self, eth_manager: 'EthereumManager'):
         super().__init__(eth_manager=eth_manager, version=3)
 
+    @cache_response_timewise()
```

Comment:
This is just caching the pool finding, right? not the actual price, since that should already be cached in the DB?

Reply:
Yes, is cache for the path not the prices. There are some common paths and this improves the price query for this oracles